From ec2ecde60a0e4d7104c8f3a68912b7e85ffc69c0 Mon Sep 17 00:00:00 2001
From: Roman Royer <romanroyer@yahoo.com>
Date: Wed, 4 May 2016 19:04:36 -0700
Subject: [PATCH 30/32] Allow sorting Applications by size

Allow sorting applications by size in Settings->Apps
just like in Settings->Storage/USB->Apps

Change-Id: I9e022162110e1feff20f257992086e66735e0f5e
---
 .../applications/manageapplications/ManageApplications.java | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/com/android/settings/applications/manageapplications/ManageApplications.java b/src/com/android/settings/applications/manageapplications/ManageApplications.java
index 51b24e1913..91ce70f4cb 100644
--- a/src/com/android/settings/applications/manageapplications/ManageApplications.java
+++ b/src/com/android/settings/applications/manageapplications/ManageApplications.java
@@ -631,9 +631,11 @@ public class ManageApplications extends InstrumentedFragment
         }
         mOptionsMenu.findItem(R.id.advanced).setVisible(false);
 
-        mOptionsMenu.findItem(R.id.sort_order_alpha).setVisible(mListType == LIST_TYPE_STORAGE
+        mOptionsMenu.findItem(R.id.sort_order_alpha).setVisible(
+                (mListType == LIST_TYPE_STORAGE || mListType == LIST_TYPE_MAIN)
                 && mSortOrder != R.id.sort_order_alpha);
-        mOptionsMenu.findItem(R.id.sort_order_size).setVisible(mListType == LIST_TYPE_STORAGE
+        mOptionsMenu.findItem(R.id.sort_order_size).setVisible(
+                (mListType == LIST_TYPE_STORAGE || mListType == LIST_TYPE_MAIN)
                 && mSortOrder != R.id.sort_order_size);
 
         mOptionsMenu.findItem(R.id.show_system).setVisible(!mShowSystem
-- 
2.17.1

